spark/v4.0/spark/src/test/java/org/apache/iceberg/spark/actions/TestRewriteTablePathsAction.java

```diff
@@ -354,8 +382,10 @@ public void testManifestRewriteAndIncrementalCopy() throws Exception {
     checkFileNum(1, 1, addedManifest, 3, postReweiteResult);
   }
 
-  @Test
-  public void testDeleteDataFile() throws Exception {
+  @ParameterizedTest
```

Comment:
rather than parameterizing every single method, we typically parameterize at the class level by using `@ExtendWith(ParameterizedTestExtension.class)`. You can take at a few other test classes that already do this, such as `TestRewritePositionDeleteFilesAction`